includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -39,6 +39,8 @@ class UPE_Payment_Gateway extends WC_Payment_Gateway_WCPay {
 
 	const METHOD_ENABLED_KEY = 'enabled';
 
+	const UPE_APPEARANCE_TRANSIENT = 'upe_appearance';
```

Comment:
```suggestion
	const UPE_APPEARANCE_TRANSIENT = 'wcpay_upe_appearance';
```
Feel free to ignore this suggestion, but any thoughts on chucking in a `wcpay` or maybe adding in the gateway ID, to make this transient value stand out a little more?